docs/user-guide/diff-strategies.md

```diff
@@ -43,6 +43,11 @@ are only triggered when:
 - There is a new revision in the repo which the Argo CD Application is
   targeting.
 - The Argo CD Application spec changed.
+- The resource already exists.
```

Comment:
Here we are listing the conditions where requests are sent to kubeapi instead of reading from the cache. I would just remove this line.